### ISSUE TITLE
Figure out communications between the Rio and Arduino for Neopixels.

### ISSUE THREAD

**Issue by wh1ter0se** [wh1ter0se/PowerUp-2018#45]
Based off #38.

The plan is to use the output pins on the Rio and the input pins on the duino. 

This is really just matter of figuring out how to manually change the voltage of a pin in WPILib.

**Comment by Nathan-2018** [comment]
would we need resistors because i don't think the duino can take more than 1 amp per pin?

**Comment by meads594** [comment]
[375275a.pdf](https://github.com/wh1ter0se/PowerUp-2018/files/1569041/375275a.pdf)
I've attached the specification sheet for the RoboRio. 

We'll have to decide if we want to use analog or digital pins for the communication between the Arduino and the RoboRio. Digital is probably easier, but may require multiple pins, depending on what we want to do with it. In principle, if we used an analog pin, we could have different voltage levels mean different things. (Although this has its own complications.) 

But, in general, you are right. If we use an analog pin then we will have to watch the amount of current to make sure we don't fry the arduino. (The RoboRio documentation says it can only drive 3mA of current, so it may not be a problem.)
